packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/upgrade-storage/style.scss

```diff
@@ -0,0 +1,8 @@
+// avoid translation menu getting below summary
+.order-summary {
+  z-index: 1009;
+}
+
+.oui-tile__description {
+  word-break: break-word;
+}
```

Comment:
**nitpick:** Please use scoped classes to avoid not anticipated changes at other places.

Reply:
Done here: https://github.com/ovh/manager/pull/8392/commits/ec34b726aa584b0c36591aee27bde3b6af0f7c95